motionplan/constraint.go

```diff
@@ -579,8 +579,12 @@ func ConstraintsFromProtobuf(pbConstraint *motionpb.Constraints) *Constraints {
 	orientConstraintFromProto := func(orientConstraints []*motionpb.OrientationConstraint) []OrientationConstraint {
 		toRet := make([]OrientationConstraint, 0, len(orientConstraints))
 		for _, orientConstraint := range orientConstraints {
+			orientTol := 0.
```

Comment:
```suggestion
			orientTol := 2.0
```
Should this be this?

Reply:
Nope, we check for and set this later.